lib/breakdown.rb

```diff
@@ -1,20 +1,34 @@
 require 'csv'
+require_relative './games'
+require_relative './teams'
+require_relative './game_teams'
 
 class Breakdown
 
-  def self.read(path)
-    tracker = {}
 
-    #sets first line of csv (headers) to keys of 'tracker' hash
-    CSV.readlines(path)[0].each do |header|
-      tracker[header.to_sym.downcase] = []
+  def self.read_games(path)
+    tracker = []
+
+    CSV.foreach(path, {:headers => true, :header_converters => :symbol}) do |row|
+      tracker << Games.new(row)
     end
+    tracker
+  end
+
+  def self.read_teams(path)
+    tracker = []
+
+    CSV.foreach(path, {:headers => true, :header_converters => :symbol}) do |row|
+      tracker << Teams.new(row)
+    end
+    tracker
+  end
+
+  def self.read_game_teams(path)
+    tracker = []
```

Comment:
This change allows us to have individual instances of Games, Teams, and GameTeams in as arrays instead of one big instance of Game like we had.